src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail_controller.js

```diff
@@ -192,6 +192,22 @@ export default class ReplicationControllerDetailController {
    * @export
    */
   hasMemoryUsage(pod) {
-    return !!pod.metrics && (!!pod.metrics.memoryUsage || pod.metrics.memoryUsage === 0);
+    return !!pod.metrics && !!pod.metrics.memoryUsageHistory && pod.metrics.memoryUsageHistory.length > 0;
+  }
+
+  /**
+   * Produces the last value of the last element of an array of MetricsResults.
+   * @param {!Array<!backendApi.MetricResult>} ts
+   * @return {number}
+   * @export
+   */
+  latestTimeseriesValue(ts) {
```

Comment:
This was an attempt to allow for deprecation of the old values, but less code is better! Fixed in d94f58ce5d174241ea7749af349ae8f435ae0834